code/modules/shuttle/shuttle_manipulator.dm

```diff
@@ -1,3 +1,4 @@
+#define PREVIEW_OR_SHUTTLE_SPAWN_COOLDOWN 2 SECONDS /// We really would rather not have admins altering this
```

Comment:
```suggestion
/// We really would rather not have admins altering this
#define PREVIEW_OR_SHUTTLE_SPAWN_COOLDOWN 2 SECONDS 
```
autodoc comments only work when you put them the line before. Can you also describe what this define does in the autodoc comment

Reply:
Done